.travis.yml

```diff
@@ -16,3 +17,11 @@ addons:
     - libenchant-dev
 git:
   depth: false
+deploy:
+  provider: pypi
+  user: Julian
+  password: ...
```

Comment:
You'll need to add your PyPI password here: https://docs.travis-ci.com/user/environment-variables/#defining-encrypted-variables-in-travisyml